xcs/delay_scan.py

```diff
@@ -10,7 +10,8 @@
 from pcdsdaq.preprocessors import daq_during_wrapper
 from pcdsdevices.interface import BaseInterface
 
-def delay_scan(daq, time_motor, time_points, sweep_time, duration=None):
+def delay_scan(daq, time_motor, time_points, sweep_time, duration=None, 
+               record=None, use_l3t=False, controls=None):
```

Comment:
I would normally assume `use_l3t=False` is correct but I remember some discussion about how they wanted their `use_l3t` argument set and don't remember the outcome. @spenc333 do you remember?

Reply:
Default False is good